Art/chrisguenther/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Snowfall</title>
+  </head>
+  <body>
+    <div class="snowflake">❄</div>
+    <div class="snowflake blurry">❄</div>
+    <div class="snowflake small">❄</div>
+    <div class="snowflake blurry small">❄</div>
+    <div class="snowflake">❄</div>
+    <div class="snowflake blurry">❄</div>
+    <div class="snowflake small">❄</div>
+    <div class="snowflake blurry small">❄</div>
+    <div class="snowflake">❄</div>
+    <div class="snowflake blurry">❄</div>
+    <div class="snowflake small blurry">❄</div>
+    <div id="moon"></div>
+    <div class="custom-shape-divider-bottom-1655954136">
+      <svg
```

Comment:
Hello @cmguenther , great work you got here but it looks like you have used svg here. As per the contribution guides, svgs may not be allowed.

![image](https://user-images.githubusercontent.com/32850163/175554564-f2e6705c-a659-47cb-954b-f65e75ffd8ba.png)